CommonTools/ParticleFlow/python/Isolation/pfPhotonIsolation_cff.py

```diff
@@ -3,8 +3,8 @@
 from RecoParticleFlow.PFProducer.photonPFIsolationDeposits_cff import *
 from RecoParticleFlow.PFProducer.photonPFIsolationValues_cff import *
 
-pfPhotonIsolationSequence = cms.Sequence(
-    photonPFIsolationDepositsSequence +
-    photonPFIsolationValuesSequence
+pfPhotonIsolationTask = cms.Task(
+    photonPFIsolationDepositsTask ,
+    photonPFIsolationValuesTask
     )
-
+pfPhotonIsolationSequence = cms.Sequence(pfPhotonIsolationTask)
```

Comment:
If one wants to clean up, one could now remove the definitions of `photonPFIsolationDepositsSequence` and `photonPFIsolationValuesSequence` from 
- RecoParticleFlow/PFProducer/python/photonPFIsolationValues_cff.py
- RecoParticleFlow/PFProducer/python/photonPFIsolationDeposits_cff.py